indexer/elastic.py

```diff
@@ -44,3 +52,23 @@ def elasticsearch_client(self) -> Elasticsearch:
 
         # Connects immediately, performs failover and retries
         return Elasticsearch(hosts.split(","))
+
+    def read_file(self, template_name: str) -> Union[dict, Any]:
+        assert self.args
+        elasticsearch_config_dir = self.args.elasticsearch_config_dir
+        file_path = os.path.join(elasticsearch_config_dir, template_name)
+        with open(file_path, "r") as file:
+            data = file.read()
+        return json.loads(data)
+
+    def index_template_data(self) -> Any:
+        json_data = self.read_file(template_name="create_index_template.json")
+        return json_data
```

Comment:
```suggestion
    def load_index_template(self) -> Any:
        return self._load_template("create_index_template.json")
```

Since `"create_index_template.json`" is expected to never change, you should define it as a constant